test/js/buffer/icon_vertex_buffer.test.js

```diff
@@ -1,16 +1,14 @@
 'use strict';
 
 var test = require('prova');
-var Buffer = require('../../../js/data/buffer/buffer');
 var IconVertexBuffer = require('../../../js/data/buffer/icon_vertex_buffer');
 
 test('IconVertexBuffer', function(t) {
-    var buf = new Buffer();
-    t.ok(new IconVertexBuffer(buf), 'default buffer');
-    var iconBuffer = new IconVertexBuffer(buf);
+    t.ok(new IconVertexBuffer(), 'default buffer');
+    var iconBuffer = new IconVertexBuffer();
     iconBuffer.setupViews();
     t.equal(iconBuffer.itemSize, 16);
-    t.equal(iconBuffer.add(0, 0, 0, 0, 0, 0, 0, 0, [0, 0], 0, 0), undefined);
+    t.equal(iconBuffer.add(0, 0, 0, 0, 0, 0, 0, 0, 0), undefined);
```

Comment:
Ideally this interface wouldn't change. Can you modify the `add` method so that this test passes as it was originally written?